### DISCUSSION TITLE
Are you also responsible for installing closed source commercial software？

### DISCUSSION THREAD

**Discussion by egege** [Homebrew/discussions#1901] [Casks]
I want to install fiddler everywhere and microsoft to do, but I can't search for it.
So to provide the installation，Is homebrew's project maintainer responsible for this？

**Comment by SMillerDev** [comment]
Homebrew is not responsible for any software you install, as defined in the license. 

You can however also use Homebrew to install closed source software on macOS, but those projects are not available in the cask repository.
